### DISCUSSION TITLE
Update the issue status in the 2023 milestone

### DISCUSSION THREAD

**Discussion by nowhszh** [variar/klogg#617] [General]
@variar The issue status in the 2023 milestone may need to be updated so that we know more clearly which issue to resolve

**Comment by variar** [comment]
Indeed, the 20xx.next milestone is more like a catch all bucket now. I'll go through the list and move some to 2024.next.

**Comment by variar** [comment]
I've updated the issues in 2023 milestone. There are already many nice improvements this year. I want to make a new stable release in a couple of months so I moved 'big features' to next milestone. There are still some fixes for 2023 that would be nice to have if implementation doesn't require big refactorings.

**Reply by nowhszh** [reply]
@variar Thanks
